tools/scripts/ansible/templates/cldrbackup/config.j2

```diff
@@ -0,0 +1,3 @@
+Host corp.unicode.org
+ User cldrbackup
```

Comment:
same here, this `cldrbackup` (on corp. not on the local system) could be a variable

Reply:
Yes. There's probably a lot of refactoring that should be done, for better encapsulation, etc.

* In Ansible, when should we use yml vs j2 vs  raw files?
* How should we organize setup-playbook.yml and other files to keep the code units short, simple, and loosely coupled?